packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/updateModal.js

```diff
@@ -31,13 +33,17 @@ const UpdateModal = props => {
     return (
         <Dialog
             confirmText={'Update'}
+            confirmTranslationId={'global.updateButton'}
```

Comment:
I think it makes sense to have the parent component do the translation and simply provide the text prop instead of text and id. Similar to the `title` prop below.

Reply:
There are some strings specified in the `defaultProps` of the Dialog component, and needed a way to translate the defaults while also supporting the ability to replace the default text